functions/src/definitions/common/counters.ts

```diff
@@ -61,12 +61,12 @@ const getVoteCounts = async function (messageRef: DocumentReference) {
     totalVoteRequestQuery,
   ])
   const totalVoteRequestsCount = voteRequestCountSnapshot.data().count ?? 0
-  const factCheckerCount = totalVoteRequestsCount - errorCount //don't count "error" votes in number of fact checkers, as this will slow the replies unnecessarily.
-  const validResponsesCount = responsesCount - errorCount //can remove in future and replace with nonErrorCount
+  const factCheckerCount = totalVoteRequestsCount - passCount //don't count "error" votes in number of fact checkers, as this will slow the replies unnecessarily.
+  const validResponsesCount = responsesCount - passCount //can remove in future and replace with nonErrorCount
```

Comment:
Similar to L63, should we include a check? Defaulting to `0`

Reply:
I think no need actually. cos L63 is the only resolved promise in the promise.all that isn't coming from the async getCount function. The getCount function will definitely return an integer >= 0. So i think its fine.